cli/man/splinter-circuit-propose.1.md

```diff
@@ -0,0 +1,187 @@
+other nodes. When the other nodes receive this proposal, they vote to accept or
+reject it with `splinter circuit vote`. If all nodes accept the proposal, the
+circuit is created.
+
+It is necessary to specify the participating nodes for a circuit proposal as well
+as information for the intended use and operation of the circuit. Circuit proposals
+may be constructed multiple ways, via command-line options using this command or
+using the `splinter circuit template` command. The `splinter circuit template`
+command offers a partially completed circuit proposal, requiring less input than
+using `splinter circuit propose`. More information on how to use circuit templates
+can be found in the splinter-circuit-template(1) man page.
+
+FLAGS
+=====
+`-n`, `--dry-run`
+: Show the circuit definition without submitting the proposal
+
+`-h`, `--help`
+: Prints help information
+
+`-q`, `--quiet`
+: Decrease verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`-V`, `--version`
+: Prints version information
+
+`-v`
+: Increases verbosity (the opposite of -q). Specify multiple times for more
+  output.
+
+OPTIONS
+=======
+`--comments <comments>`
+: Adds human-readable comments to the circuit proposal.
+
+`-k, --key <private-key-file>`
+: Specifies the full path to the private key file.
+
+`--management <management_type>`
+: Specifies the circuit management type. Circuit management type indicates the
+  application authorization handler which handles the circuit’s change proposals.
+
+`--metadata <application_metadata>` ...
+: Provides application-specific metadata for the circuit proposal. Repeat this
+  option to provide multiple entries for the application metadata.
+
+`--metadata-encoding <metadata_encoding>`
+: Sets the encoding type for the application metadata (default: `string`).
+  Accepted values: `json`, `string`.
+
+`--node <node_string>` ...
+: Specifies a node that should be part of the circuit, using the format
+  `<node_id>::<endpoint>`. This node ID must match the node ID and endpoint entry
+  in the node registry. The proposer must also specify its own node, if it is to
+  be included on the circuit proposal. Repeat this option to specify multiple
+  nodes.
+
+`--service <service_string>` ...
+: Specifies the service ID and allowed nodes, using the format
+  `<service-id>::<allowed_nodes>`. The <allowed_nodes> specifies the node which
+  the service will run on, currently only one node ID is allowed.
+
+`--service-arg <service_argument>` ...
+: Passes key/value arguments to the specified service (as defined by
+  `--service`), using the format `<service_id>::<key>=<value>`. Service arguments
+  provided must match those required to create the service. The glob operator,
+  `*`, may be used in place of the <service_id> to match all or certain parts
+  of the <service_id>. For instance, `alpha*::<key>=<value>` to match all service
```

Comment:
`alpha` is not a valid `service_id`. Service IDs have a max of 4 char in length and must be ASCII alphanumeric characters.   All the examples of service IDs in the man page should show service IDs in the correct format. This is a good example to look at for this case
https://github.com/Cargill/splinter/blob/master/cli/src/main.rs#L50-L54